### ISSUE TITLE
Bug: rounding's collision

### ISSUE THREAD

**Issue by fedoranvar** [hyprwm/Hyprland#542]
Good day to you!

I have settings for default rounding, an have rules for some windows

But i have collision for that class of windows
![image](https://user-images.githubusercontent.com/52028706/185364555-9f17c0a1-82aa-4404-8550-dfea050fd35a.png)

**Comment by vaxerski** [comment]
can't reproduce:
![image](https://user-images.githubusercontent.com/43317083/185445596-fb1c0f3f-a2f4-4ea2-8663-45b5d5d51a1e.png)

config?

**Comment by fedoranvar** [comment]
<details>
<summary>config</summary>
```
# ---------------------------
#           general
# ---------------------------

$monL=DP-1
$monR=HDMI-A-1


monitor=$monL,1280x1024@60,0x0,1
monitor=$monR,1280x1024@60,1281x0,1

workspace=$monL,name:term
workspace=$monR,name:web

input {
    kb_layout=us,ru
    kb_options = grp:win_space_toggle
    kb_rules=evdev
    follow_mouse=1
    natural_scroll=1
}

general {
    sensitivity=0.9
    main_mod=SUPER

    layout=dwindle
    gaps_in=5
    gaps_out=10
    border_size=1
    col.active_border=0xfae9aa82
    col.inactive_border=0x00000000
    # cursor_inactive_timeout=2
    damage_tracking=none
    no_border_on_floating=1
}


# bezier=overshot,0.42,0,0.58,1
animations {
    enabled=1
}

animation=windows,1,3,default,popin 30%
animation=workspaces,1,3,default,fade
animation=fadeSwitch,1,4,default
animation=border,1,5,default

dwindle {
    pseudotile=1 # enable pseudotiling on dwindle
    special_scale_factor=0.95
    preserve_split=1
    force_split=2
}

misc {
    disable_hyprland_logo=1
    mouse_move_enables_dpms=1
}

input:touchpad {
    disable_while_typing=1
    natural_scroll=1
}

gestures {
    workspace_swipe=1
    workspace_swipe_fingers=2
    workspace_swipe_distance=50
}


#### DECORATIONS 
#
decoration {
    rounding=25
    multisample_edges=1
    active_opacity=0.97
    inactive_opacity=0.92
    fullscreen_opacity=1
    blur=0
}
windowrule=rounding 0,util
```
</details>

**Comment by vaxerski** [comment]
Is that maybe the shadow? try turning off shadows.

**Comment by fedoranvar** [comment]
yes, turned off shadows and problem solved

thank you!

**Comment by fedoranvar** [comment]
but then comes a question:

is it normal that shadows didn't respect ruled roundings?

**Comment by vaxerski** [comment]
no, it's a bug, and now that i look at my screenshot i can also see it

**Comment by vaxerski** [comment]
fixed in 6cae44e2c0254e959ad05e4f2d98576cb6b82258

**Comment by fedoranvar** [comment]
Thank you!
